locale/uk/get-involved/development.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Розробка
+title: Development
```

Comment:
Why this has been changed back to English?

Reply:
I had though 'uk' meant united kingdom English.  But sounds like it is actually Ukrainian.  Sorry will remove that change.